### ISSUE TITLE
How to delete messages?

### ISSUE THREAD

**Issue by rc0n9027** [lutzseverino/discord-books#5]
```
JDAReceivable jdaReceivable = new JDAReceivable(e.getChannel());
jdaReceivable.receive(page);
```

Is there a way to delete the message executed by writing the code above?

ex) msg.delete().queue();

**Comment by lutzseverino** [comment]
There's no `#receive` method that returns the `MessageAction` (to be included in 2.7.1), the only way to remove the message would be to manually search for it. What exactly do you want to do?

I also speak Spanish, btw.

**Comment by rc0n9027** [comment]
> There's no `#receive` method that returns the `MessageAction` (to be included in 2.7.1), the only way to remove the message would be to manually search for it. What exactly do you want to do?
> 
> I also speak Spanish, btw.

When a button other than the next and previous buttons is pressed, I want to trigger the event of that button.
Like JDA EventWaiter.
Also, when the button is clicked, I want to delete the message executed by receive.
Do you know how to manually delete it?

ex) I fetched people's nicknames from DB and made it into a button.
However, when I press the next and previous buttons, it jumps to the next event and throws an error.

**Comment by lutzseverino** [comment]
This is unrelated to this library. I'll help you out and close the issue.

Buttons cannot be acknowledged more than once, if you're listening to a button interaction you **will** have to filter out every button that contains an ID starting with `book:`, this is not the library's fault, it's just how Discord works.

I suspect you don't know how events work on JDA, read the related [wiki page](https://jda.wiki/introduction/events/).

If you want every button to delete its message on click, but avoid acknowledging twice, the simplest thing you can do is this:
```java
public class Foo extends ListenerAdapter {
    @Override public void onButtonInteraction(ButtonInteractionEvent event) {
        if (event.getComponentId().startsWith("book:")) return;

        event.getMessage().delete().queue();
    }
}
